helpers.rb

```diff
@@ -72,18 +72,24 @@ def check_password(password_hash, password)
         correct_pass.is_password?(password)
       end
 
+      def update_password(profileId, newPassword)
```

Comment:
I would prefer to keep an actual update of user data in the controller, since we currently do all of our model manipulation within the controllers.

Reply:
I think this sounds fine. In addressing some of the comments about superadmin access below, I changed this helper function to return the hash instead of modifying the Profile in the helper function. The database update will now be on the route controller.